src/ui/ProgressCircle.scss

```diff
@@ -7,6 +7,7 @@
 .widget-progress-circle {
     .h1, .h2, .h3, .h4, .h5, .h6 {
         margin: 0px;
+        height: 10em;

```

Comment:
The circle size does not actually match with that of other browsers. Maybe increase to 15em.
However, this solution breaks on the mobile. it displays the text and size of the circle.
Probably you might have to it only to IE. Not sure how by css or in code